src/main/java/com/textadventure/Lexer.java

```diff
@@ -41,10 +38,9 @@ private TokenType(String pattern) {
 			} else if (matcher.group(TokenType.DIRECTION.name()) != null) {
 				tokens.add(new Token(TokenType.DIRECTION, matcher.group(TokenType.DIRECTION.name())));
 				continue;
```

Comment:
End of loop and end of if statements there is no need to have **continue** here.